package.json

```diff
@@ -22,18 +22,18 @@
   "dependencies": {
     "@expo-google-fonts/roboto": "~0.1.0",
     "@expo-google-fonts/rubik": "~0.1.0",
-    "@ory/client": "0.0.1-alpha.19",
-    "@ory/kratos-client": "0.0.0-next.059ed9c9cacb",
+    "@ory/client": "^0.0.0-next.51d4b3802651",
     "@ory/themes": "0.0.21",
     "@react-native-async-storage/async-storage": "~1.15.0",
     "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/native": "~5.8.10",
     "@react-navigation/stack": "~5.12.8",
     "@sentry/integrations": "~6.11.0",
     "@types/node": "~14.14.10",
-    "axios": "~0.21.0",
+    "axios": "~0.21.1",
     "expo": "~42.0.0",
     "expo-application": "~3.2.0",
+    "expo-cli": "^6.0.5",
```

Comment:
As far as I know this should instead be installed globally `npm i -g expo-cli`, if that is the case please remove it here

Reply:
True, I guess `npx` added it.

Reply:
Hm, it was just re-added by the `test/e2e/run.sh` script in kratos, as it points to the local repo. 

We might want to take a look at that later. For now I'll check that I don't commit it here.